File: codewars/8kyu/return-negative.py
# SOLUTION
# ==============================================================================
def make_negative(number):
    return number if number <= 0 else -number

# TESTS
# ==============================================================================
if __name__ == "__main__":
    assert make_negative(1) == -1, "Test case 1 failed"
    assert make_negative(-5) == -5, "Test case 2 failed"
    assert make_negative(0) == 0, "Test case 3 failed"
    assert make_negative(42) == -42, "Test case 4 failed"
    print("All test cases passed!")

File: codewars/8kyu/return-negative.cpp
#include <iostream>
#include <cassert>

int makeNegative(int num) {
    return num > 0 ? num * -1 : num;
}

void runTests() {
    assert(makeNegative(1) == -1);
    assert(makeNegative(-5) == -5);
    assert(makeNegative(0) == 0);
    std::cout << "All tests passed!" << '\n';
}

int main() {
    runTests();
    return 0;
}
